src/main/resources/institutional-config.properties

```diff
@@ -15,6 +15,7 @@ dailyEmail.cronExpression=0 0 1 * * ?
 weeklyEmail.cronExpression=0 0 2 ? * 2
 periodic.thread.dump=false
 xml.pipeline.lifecycle.enabled=false
+email.reminder.lifecycle.enabled=false
```

Comment:
It would be good to add a comment about why this was set to false and that it should be removed when we upgrade to 4/1/21 version of financials.

Reply:
Ok, I'll add a comment accordingly.